### ISSUE TITLE
Url rewrite not working same product name problem

### ISSUE THREAD

**Issue by hoodiegamerss** [faktiva/prestashop-clean-urls#84]
Hello,

I have a problem with same name products. 

https://www.mydomain.com/red-car-1 
https://www.mydomain.com/red-car-2

my product url is {rewrite}-{id}

when I click the www.mydomain.com/red-car-2 it gives an error like;

[Debug] This page has moved
Please use the following URL instead: https://www.mydomain.com/red-car-1

But those 2 product is different just their name is same.

Prestashop version is 1.6.1.12

**Comment by drAlberT** [comment]
if you need to use the ID in your URL you don't need this module .. Am I wrong?  Possibly give a real live example to navigate to and see the issue

**Comment by hoodiegamerss** [comment]
I just want to use it for categories and cms pages. Not for products.

**Comment by hoodiegamerss** [comment]
For example this two product have a same name with different ID numbers
https://www.eurodiamond.com/051-karat-tektas-pirlanta-yuzuk-14

https://www.eurodiamond.com/051-karat-tektas-pirlanta-yuzuk-26
when i click to this product it gives an error like
[Debug] This page has moved
Please use the following URL instead: https://www.eurodiamond.com/051-karat-tektas-pirlanta-yuzuk-14

**Comment by drAlberT** [comment]
ok, got the point .. will inspect ASAP. Thanks for reporting

**Comment by hoodiegamerss** [comment]
Module does not read the ID of the products. I have 300 products. I'm in trouble right now :(

**Comment by drAlberT** [comment]
uninstall this module and your IDs will be read :)

**Comment by hoodiegamerss** [comment]
I uninstalled nothing has changed

**Comment by drAlberT** [comment]
cache cleared?

**Comment by hoodiegamerss** [comment]
Yes cleared. Your module is not working for products which have same name also with ID.

**Comment by drAlberT** [comment]
I got this .. but you are saying you have troubles even if you uninstall the module ...

**Comment by hoodiegamerss** [comment]
Yes I have trouble. I dont know what should I do now. I bought Pretty Urls but still can not access my products.
